### PR TITLE
Native RecyclerView sample

### PR DESCRIPTION
**Pull request by irwansetiawan**
Add Native RecyclerView sample on Standalone module

### CODE REVIEW

File: standalone/src/main/java/com/criteo/publisher/samples/standalone/ContentItem.java [hunk 1]
```diff
@@ -0,0 +1,15 @@
+package com.criteo.publisher.samples.standalone;
+
+public class ContentItem {
```
Comment (Amokrane) [inline standalone/src/main/java/com/criteo/publisher/samples/standalone/ContentItem.java:19]: So-called value object classes, such as this one, should be immutable. Immutability helps with thread safety and debugging as the value of an object doesn't change anymore after its creation (more info: https://dev.to/fragmented/075-effective-java--item-15). 
To achieve immutability, you need to make all attributes final.

```
public class ContentItem {
   private final String title;
   private final String description;
   private final String imageUrl;
  // constructor
}
```

Also, fields should be private.

File: standalone/src/main/java/com/criteo/publisher/samples/standalone/NativeRecyclerViewActivity.java [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+import com.squareup.picasso.Picasso;
+import java.util.ArrayList;
+import java.util.List;
+
+public class NativeRecyclerViewActivity extends AppCompatActivity {
+
+  private MyAdapter adapter;
+  private CriteoNativeLoader nativeLoader;
+
+  private int dummyContentCount = 12;
```
Comment (Amokrane) [inline standalone/src/main/java/com/criteo/publisher/samples/standalone/NativeRecyclerViewActivity.java]: Both  `dummyContentCount` and `nativeAdPosition` should be constants:
```
private static final int DUMMY_CONTENT_COUNT = 12;
private static final int NATIVE_AD_POSITION = 3;
```

File: standalone/src/main/java/com/criteo/publisher/samples/standalone/NativeRecyclerViewActivity.java [hunk 3]
```diff
@@ -0,0 +1,168 @@
...
+    super.onCreate(savedInstanceState);
+    setContentView(R.layout.activity_native_recycler_view);
+
+    adapter = new MyAdapter();
+    RecyclerView recyclerView = findViewById(R.id.recyclerView);
+    recyclerView.setLayoutManager(new LinearLayoutManager(this));
+    recyclerView.setAdapter(adapter);
+
+    // add some dummy content
+    for (int i=0; i<dummyContentCount; i++) {
```
Comment (Amokrane) [inline standalone/src/main/java/com/criteo/publisher/samples/standalone/NativeRecyclerViewActivity.java]: Are you using any formatter? I recommend using Criteo's Java code style (will send you the link).

What triggered this question is the lack of spacing around the `<` operator.

**Review by Amokrane — CHANGES_REQUESTED** [review]

File: standalone/src/main/java/com/criteo/publisher/samples/standalone/ContentItem.java (hunk 1 above)
Comment (irwansetiawan) [reply standalone/src/main/java/com/criteo/publisher/samples/standalone/ContentItem.java:19]: Done, thanks!

File: standalone/src/main/java/com/criteo/publisher/samples/standalone/NativeRecyclerViewActivity.java (hunk 2 above)
Comment (irwansetiawan) [reply standalone/src/main/java/com/criteo/publisher/samples/standalone/NativeRecyclerViewActivity.java]: Done.

File: standalone/src/main/java/com/criteo/publisher/samples/standalone/NativeRecyclerViewActivity.java (hunk 3 above)
Comment (irwansetiawan) [reply standalone/src/main/java/com/criteo/publisher/samples/standalone/NativeRecyclerViewActivity.java]: I use Criteo formatter, but I didn't run them all the time. Have reformatted the code, thanks for reminding!

**Review by Amokrane — APPROVED** [review]

**Pull request merged**